src/cmd/services/m3coordinator/ingest/carbon/ingest.go

```diff
@@ -228,47 +246,55 @@ func (i *ingester) write(
 					zap.Any("mappingRules", rule.mappingRules),
 					zap.Any("storagePolicies", rule.storagePolicies))
 			}
+
 			// Break because we only want to apply one rule per metric based on which
 			// ever one matches first.
-			break
-		}
-	}
+			err := i.writeWithOptions(ctx, resources, timestamp, value,
```

Comment:
Could throw these writes into a consumer queue if rule.rule.Continue is true? Don't know if it's necessary atm